kasa/smart/smartmodule.py

```diff
@@ -53,14 +53,19 @@ def call(self, method, params=None):
     def data(self):
         """Return response data for the module.
 
-        If module performs only a single query, the resulting response is unwrapped.
+        If the module performs only a single query, the resulting response is unwrapped.
+        If the module does not define a query, this property returns a reference
+        to the main "get_device_info" response.
         """
+        dev = self._device
         q = self.query()
+
+        if not q:
+            return dev.internal_state["get_device_info"]
```

Comment:
If a module reports its query as non-true, it means that the information is stored in the main get_device_info responses.